src/components/DocumentationTopic.vue

```diff
@@ -300,6 +301,22 @@ export default {
     ),
     shouldShowLanguageSwitcher: ({ objcPath, swiftPath }) => objcPath && swiftPath,
     hideSummary: () => getSetting(['features', 'docs', 'summary', 'hide'], false),
+    enableHero: ({ symbolKind }) => (symbolKind ? (symbolKind === 'module') : true),
+    heroTitleStyle({ enableHero }) {
+      return enableHero ? {
+        '--hero-eyebrow': 'light-color(figure-gray-secondary)',
+        '--hero-title': 'light-color(fill)',
+      } : {
+        '--hero-eyebrow': 'var(--colors-secondary-label,var(--color-secondary-label))',
+      };
+    },
+    borderWidth({ enableHero }) {
+      return enableHero ? {
+        '--border-width': '0px',
```

Comment:
I honestly dont know if its better to have a class to turn on/off the separator or a css var like this.

Reply:
The reason why I chose to use a CSS var like this is because the border is applied in the `primary content` component. Whether or not the hero should be "enabled" (or rather have the darker background) is not available in that component. So I would need to pass in a separate prop. 
I'm also not sure which way is better. Maybe Marcus can comment on this as well?

Reply:
The only change this component needs to make (for removing dark background hero from symbol pages) is this border-width change. Versus in the `Documentation Hero` component, there is more logic so that adding this boolean prop makes sense.